src/exporter.js

```diff
@@ -0,0 +1,174 @@
+const sha1 = require('sha1');
+
+export default class {
+  constructor(deckName) {
```

Comment:
sql or db ? 

and I'm confused. Before you say to move dependencies into class. Now you say provide them on instantiating.